orchagent/portsorch.cpp

```diff
@@ -3130,6 +3130,37 @@ void PortsOrch::doTask(Consumer &consumer)
     }
 }
 
+bool PortsOrch::flushFDBEntries(sai_object_id_t bridge_port_id)
+{
+    vector<sai_attribute_t>    attrs;
+    sai_attribute_t            attr;
+    sai_status_t               rv = SAI_STATUS_SUCCESS;
+
+    SWSS_LOG_ENTER();
+
+    if (SAI_NULL_OBJECT_ID == bridge_port_id)
+    {
+        SWSS_LOG_WARN("Couldn't flush FDB. Bridge port OID: 0x%" PRIx64,
+                      bridge_port_id);
```

Comment:
if it is null object id how can you print it?

Reply:
changed to port.m_alias.c_str() instead of bridge_port_id. But, printing bridge_port_id was part of fdborch.cpp. That is how it was done there. Now, since we move to printing port.m_alias.c_str(), it will display the port name say, Ethernet120 for example.